examples/geometry/new_caching_tutorial.py

```diff
@@ -0,0 +1,265 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
```

Comment:
The old geometry tutorial works fine now; do we need another tutorial or is this just a testing area for the purposes of this PR?